internal/btf/btf_test.go

```diff
@@ -110,6 +107,20 @@ func TestParseVmlinux(t *testing.T) {
 	}
 }
 
+func BenchmarkParseVmlinux(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		spec, err := parseVmLinux(b)
```

Comment:
This benchmarks gzip decompression as well, which doesn't make sense. This should decompress first and only call loadRawSpec in the loop.

Reply:
Good point!
```
# with gzip
BenchmarkParseVmlinux-16    	      14	 123495986 ns/op
# without gzip
BenchmarkParseVmlinux-16    	      14	  88655530 ns/op
```